cves/CVE-2016-2213.yml

```diff
@@ -296,20 +322,26 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
-    note:
+    applies: true
+    note: |
+      This mechanism could fail, but the entire subsystem still needs to work. 
+      That is why a layered based approach would absolutely be useful here.
   least_privilege:
-    applies:
-    note:
+    applies: true
+    note: |
+      The user is given the ability to upload their own JPEG, which means that
+      someone could upload a malicious file to mess with the system. Restricting
+      a users ability to upload non-approved files attributed to this problem
```

Comment:
I think it's fairly common in FFMPEG for user file uploads, and thus I think the issue is more not distrusting the input enough.